apis/v1alpha2/gateway_types.go

```diff
@@ -718,8 +711,11 @@ const (
 	// interoperability.
 	ListenerConditionDetached ListenerConditionType = "Detached"
 
-	// This reason is used with the "Detached" condition when the
-	// Listener requests a port that cannot be used on the Gateway.
+	// This reason is used with the "Detached" condition when the Listener
+	// requests a port that cannot be used on the Gateway. This reason could be
+	// used in a number of instances, including:
```

Comment:
Need a newline here to render the markdown bulleted list.

Reply:
Updated, thanks!